index.js

```diff
@@ -142,6 +148,12 @@ if (process.stdout.isTTY) {
   .then(() => {
     cache.setSync("linted-hash", commitHash);
     debug('Cached Current Commit Hash');
+    if (options.verbose) {
```

Comment:
It would be great if this line also check the `options.output` array as the log can be ignored when there is no values in the array.

Reply:
Sure 👍

Reply:
This has been handled. Please verify the changes.